ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-v0.6.0.yaml

```diff
@@ -0,0 +1,151 @@
+tag_specification:
+  name: '4.0'
+  namespace: ocp
+promotion:
+  cluster: https://api.ci.openshift.org
+  namespace: openshift
+  name: knative-v0.6.0
+base_images:
+  base:
+    name: '4.0'
+    namespace: ocp
+    tag: base
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+canonical_go_repository: github.com/knative/eventing
+binary_build_commands: make install
+test_binary_build_commands: make test-install
+tests:
+- as: e2e
+  commands: "INTERNAL_REGISTRY=image-registry.openshift-image-registry.svc:5000 ENABLE_ADMISSION_WEBHOOKS=false make test-e2e"
```

Comment:
The ENABLE_ADMISSION_WEBHOOKS is not necessary. The test script doesn't have this option anymore. Also, it would be good to change the value of the INTERNAL_REGISTRY directly in the test script and remove it from here. We target OpenShift 4.0 by default.

Reply:
good point - I guess same is true for the serving bits:

https://github.com/openshift/release/blob/e15ca0593ea5865b209b8a2252d4fce5c2fe1c86/ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-v0.5.1.yaml#L21

/cc @markusthoemmes